tests/lib/utils

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
I think it's better under tests, and then it makes sense to have a lib subdirectory. This is also the approach in Rose and cylc.